src/Pact/Analyze/Check.hs

```diff
@@ -452,12 +454,12 @@ verifyFunctionProperty
   -> Text
   -> CheckableType
   -> Located Check
-  -> IO (Either CheckFailure CheckSuccess)
+  -> IO [Either CheckFailure CheckSuccess]
 verifyFunctionProperty (CheckEnv tables _consts _propDefs moduleData caps gov)
   (FunData funInfo pactArgs body) funName checkType
-  (Located propInfo check) = runExceptT $ do
+  (Located propInfo check) = fmap sequence' $ runExceptT $ do
     let modName = moduleDefName (_mdModule moduleData)
-    (args, stepChoices, tm, graph) <- hoist generalize $
+    (args, stepChoices, tm, graph, warnings) <- hoist generalize $
```

Comment:
At this point, it might be good to consider https://hackage.haskell.org/package/validation cuz this sequencing gets a bit grody. That'd be something to track for future iterations.

Reply:
👍